Modix.Bot/Modules/FunModule.cs

```diff
@@ -134,9 +134,15 @@ public async Task OwoifyAsync([Remainder][Summary("The message to owoify.")] str
             owoMessage = Regex.Replace(owoMessage, "N([aeiou])", "Ny$1");
             owoMessage = Regex.Replace(owoMessage, "N([AEIOU])", "Ny$1");
             owoMessage = Regex.Replace(owoMessage, "ove", "uv");
-            owoMessage = Regex.Replace(owoMessage, "\\!+", " " + _owoFaces[new Random().Next(0, _owoFaces.Length - 1)] + " ");
-
-            await ReplyAsync(owoMessage);
+            owoMessage = Regex.Replace(owoMessage, "(?<!\\@)\\!+", " " + _owoFaces[new Random().Next(_owoFaces.Length)] + " ");
+
+            await ReplyAsync(embed: new EmbedBuilder()
+                .WithDescription(owoMessage)
+                .WithAuthor(Context.Message.Author)
```

Comment:
We've been using our own extension method for this typically; the built-in `WithAuthor` doesn't handle users with the default Discord avatar properly

```suggestion
                .WithUserAsAuthor(Context.User)
```